tensorflow/python/ops/sparse_ops.py

```diff
@@ -2732,6 +2733,99 @@ def sparse_transpose(sp_input, perm=None, name=None):
     return transposed_st
 
 
+@tf_export("sparse.map_values")
+@dispatch.add_dispatch_support
+def map_values(op, *args, **kwargs):
+  """Applies `op` to the values of one or more `SparseTensor`s.
```

Comment:
For a little extra clarity, maybe say: "... to the nondefault values ..." or "... to the explicit values ..." or something along those lines.

If you want to be even more explicit, you could also change the text below to say something like "Replaces any `SparseTensor` in `args` or `kwargs` with its `values` tensor (which contains the non-default values for the SparseTensor), and then calls `op`."